openhands_resolver/send_pull_request.py

```diff
@@ -301,9 +301,18 @@ def update_existing_pull_request(
     pr_url = f"https://github.com/{github_issue.owner}/{github_issue.repo}/pull/{github_issue.number}"
     print(f"Updated pull request {pr_url} with new patches.")
 
-
-    # Send PR Comments
-    # TODO: run a summary of all comment success indicators for PR message
+    # Generate a summary of all comment success indicators for PR message
+    if not comment_message and additional_message:
+        try:
+            explanations = json.loads(additional_message)
+            if explanations:
+                comment_message = "OpenHands has updated the PR with the following changes:\n\n"
+                for explanation in explanations:
+                    comment_message += f"- {explanation}\n"
```

Comment:
After this message is posted, it should be summarized using a language model to make it more concise. To call the language model, we can use the same `litellm.completion()` call as is used in the `guess_success()` function with a different prompt specifically for summarization.